cmd-gen.sh

```diff
@@ -0,0 +1,15 @@
+git clone https://github.com/TheThingsNetwork/lorawan-stack.git lorawan-stack
+
+cd lorawan-stack
+
+make init
+go run ./cmd/ttn-lw-cli gen-md-doc -o ../doc/content/ttn-lw-cli
+
+# Command End Device Templates has two parents and no doc is generated, prune
+sed -i.bak 's/end-devices_templates/templates/g' ../doc/content/ttn-lw-cli/ttn-lw-cli_end-devices.md
```

Comment:
`sed -i` works differently on macOS/Linux/Windows, so can we try to do this differently?